src/ensembl/src/shared/components/layout/DocLayout.scss

```diff
@@ -0,0 +1,56 @@
+@import 'src/styles/common';
+
+$sidebarContentWidth: 320px;
+
+.DocLayout {
+  height: 100%;
+  min-height: 550px; // based on the current interface of genome browser
+  overflow: hidden;
+}
+
+.mainWrapper {
+  height: 100%;
+  position: relative;
+}
+
+.main {
+  height: 100%;
+  margin-left: $sidebarContentWidth;
+  margin-right: 48px;
+}
```

Comment:
The styles need reworking.

What we seem to want (I'm not sure, given lack of updated designs) is a CSS grid with two columns that looks like this:

![Untitled Diagram](https://user-images.githubusercontent.com/6834224/76011498-9575e800-5f0c-11ea-99bc-133d08ac9359.png)

What is currently going on in the layout is this:

- main has a left margin
- sidebar wrapper is absolutely positioned in main and spans all width and height of the viewportt
- sidebar is sitting in the left corner of sidebar wrapper

![Untitled Diagram (3)](https://user-images.githubusercontent.com/6834224/76011964-5dbb7000-5f0d-11ea-94a4-53f2a6d95160.png)

After refactoring:
- main shouldn't have a left margin
- there's no need for sidebar wrapper
- there should be a common wrapper with CSS `display: grid` and two columns: one for sidebar, another for main.

Reply:
Yes, we need to confirm the details of the design with Andrea, but she's busy with more important pages right now.

Reply:
Changed to use CSS grid.